api/controllers/v1/jobs.ts

```diff
@@ -222,3 +228,27 @@ async function NotifyBuildProgress(jobId: string): Promise<any> {
     statusCode: 200,
   };
 }
+
+function getMongoClient(config: IConfig): mongodb.MongoClient {
```

Comment:
We initialize a `MongoClient` in a few different places using the same connection string. might be worth consolidating the logic into a single helper function

Reply:
Yeah I think that's a good idea, but should probably be in a different PR.

Most of the initializations happen with `new mongodb.MongoClient(c.get('dbUrl'))`.  I could have used that here as well, but I'm leaning towards building the the connection url using the individual parts (host, username, password, e tc). We may also want to consider explicitly closing the connection in code.